web/src/components/core/InstallButton.jsx

```diff
@@ -63,11 +57,14 @@ according to the provided installation settings.") }
         </p>
       </div>
       <Popup.Actions>
-        <Popup.Confirm onClick={onAccept}>
+        <Popup.PrimaryAction key="cancel" onClick={onClose} autoFocus>
+          {/* TRANSLATORS: button label */}
+          {_("Cancel")}
+        </Popup.PrimaryAction>
+        <Popup.SecondaryAction key="confirm" onClick={onAccept}>
```

Comment:
I think I didn't explain well this. `Cancel` should be the primary, autofocused action only if there are issues. So, a bit more complicated than this.

```jsx

const Cancel = hasIssues ? Popup.PrimaryAction : Popup.SecondaryAction;
const Continue = hasissues ? Popup.SecondaryAction : Popup.PrimaryAction;

...

<Cancel key="cancel" onClick={onClose} autoFocus={hasIssues}>...</Cancel>
<Confirm key="confirm" onClick={onAccept} autoFocus={!hasIssues}>...</Confirm>